broadway/api/utils/run.py

```diff
@@ -33,7 +33,7 @@ def continue_grading_run(settings, grading_run):
                 assignment.pre_processing_pipeline,
                 GradingJobType.PRE_PROCESSING,
             )
-            queue.put(next_job)
+            queue.push(grading_run.course_id, next_job)
```

Comment:
You can retrieve the `course_id` from the `assignment` object already available above (defined on line 17)